doc-content/enterprise-only/bpmn/bpmn-tasks-overview-con.adoc

```diff
@@ -1,38 +1,69 @@
 [id='bpmn-tasks-overview-con']
-= Tasks
+= BPMN2 tasks in process designer 
+A task is an automatic activity that is defined in the process model and the smallest unit of work in a process flow. The following task types defined in the BPMN2 specification are available in the {PRODUCT} process designer palette:
 
-A task is the smallest unit of work in a process flow. {PRODUCT} uses the BPMN guidelines to separate tasks based on the types of inherent behavior that the tasks represent. A task is an automatic activity that is defined in the process model. The following tasks are available in the {PRODUCT} process designer palette:
+* Business rule tasks: Used to execute a set of business rules through the rule flow group
+* Script tasks: Used to execute a piece of code written in Java, JavaScript, or MVEL
+* User tasks: Used to include human actions as input to the business process
 
-* Business rule tasks 
-* Script tasks 
-* User tasks 
+.Task 
+[cols="2"]
+|===
+| Business rule task
+| image:BPMN2/bpmn-business-rule-task.png[]
+
+| Script task
+| image:BPMN2/bpmn-script-task.png[]
 
-image::BPMN2/bpmn-tasks-designer.png[]
+| User task
+| image:BPMN2/bpmn-user-task.png[]
 
-The following service tasks are also available:
+|===
 
-* Service tasks
-* Decision tasks 
+In additions, the BPMN2 specification provides the ability to create custom tasks. The following predefined custom tasks are included with {PRODUCT}: 
 
+* Rest service tasks: Used to invoke a remote RESTful service
+* Email service tasks: Used to send an email
+* Log service tasks: Used to log a message
+* WebService service tasks: Used to invoke a remote WebService call
+* DecisionTask tasks: Used to execute a DMN diagram
+
+
+.Business rule task 
+A business rule task defines a set of rules that are evaluated and fired on task execution. Any rule defined as part of the rule flow group in a rule resource is fired.
+
+image::BPMN2/bpmn-business-rule-task.png[]
+
+
+When a process reaches a business rule task, the engine starts executing the rules with the defined rule flow group.
+When there are no more active rules with the rule flow group, the execution continues to the next element.
+During the rule flow group execution, new activations belonging to the active rule flow group can be added to the agenda because these acativations are changed by other rules.
+
+.Script task
+A script task represents a script to be executed during the process execution.
+
+image::BPMN2/bpmn-script-task.png[]
+
+
+
+The associated script can access process variables and global variables. When using a script task:
```

Comment:
"When using a script task:" should technically be a complete sentence, per IBM. Also, make the listed items parallel. Should be Avoid, Ensure that scripts are executed, Avoid, Ensure that scripts do not throw. Or similar.